index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header>
+    <header id="navigation">
+        <a href="#" class="logo">My Logo</a>
+        <nav class="large clearfix">
+            <i class="fa fa-bars"></i>
+            <ul>
+              <li>
+                <a href="#">Portfolio</a>
+              </li>
+              <li>
+                <a href="#">About</a>
+              </li>
+              <li>
+                <a href="#">Contact</a>
+              </li>
+            </ul>
+            <div class="humburger">
+                <a href="#nav">&equiv;</a>
+            </div>
+          </nav>
+    </header>
+    </header>
+    <section class="hero-section">
+        <div class="hero-container">
+            <h1 id="name">
+                I’m Ojwang. Glad to see you!
+            </h1>
+            <p class="details">
+                I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
+            </p>
+            <a href="#" class="connect">let's connect</a>
+            <ul class="socials">
```

Comment:
- [ ] I noticed that your paragraph didn't reflect the figma design paragraph , and aslo the `LET'S CONNECT` should be in uppercase , kindly fix this to reflect the original design 
Your design
![Screenshot from 2023-04-27 20-54-36](https://user-images.githubusercontent.com/88894525/234965457-1813745e-1a9c-485b-be15-8f59faffa410.png)

Figma design 
![Screenshot from 2023-04-27 20-58-20](https://user-images.githubusercontent.com/88894525/234965654-5e049adf-07ca-4def-97b5-34ec84bdd37e.png)